WordPress/Classes/ViewRelated/Activity/RewindStatus+multiSite.swift

```diff
@@ -0,0 +1,16 @@
+import WordPressKit
+
+extension RewindStatus {
+    func isMultiSite() -> Bool {
```

Comment:
[nit] JN uses "multisite", not "multiSite"

```suggestion
    func isMultisite() -> Bool {
```

Reply:
Done in 7c771275ddbc78fc180aa7e242e98fd3dc91136b